proxy/src/http/transaction.rs

```diff
@@ -173,17 +226,20 @@ mod test {
         let registry = registry::Registry::new(radicle_registry_client::Client::new_emulator());
         let store = kv::Store::new(kv::Config::new(tmp_dir.path().join("store"))).unwrap();
         let cache = registry::Cacher::new(registry, &store);
+        let now = registry::Timestamp::now();
 
         let tx = registry::Transaction {
             id: registry::Hash(radicle_registry_client::TxHash::random()),
             messages: vec![registry::Message::ProjectRegistration {
                 project_name: registry::ProjectName::try_from("upstream").unwrap(),
                 org_id: registry::Id::try_from("radicle").unwrap(),
             }],
-            state: registry::State::Applied {
-                block: registry::Hash(radicle_registry_client::Hash::random()),
+            state: registry::State::Confirmed {
+                block: 1,
+                progress: 1,
```

Comment:
According to the API docs, this would start at `0`. However if this is the number of confirmations, this is correct, and the docs are wrong.